content/content-strategy/identify-business-needs/who-to-talk/index.yml

```diff
@@ -0,0 +1,19 @@
+pagetitle: Who to talk to
+description: You’ll need to look at documents, run workshops and speak to different business areas.
+weight: 10
+layout: layout/topic
+theme: blue
+header:
+  - /_shared/globalheader.md
+  - /_shared/header.md
+main:
+  - intro.md
+  - heading.md
```

Comment:
Missing table of contents partial. There should be one here given the structure of the page and inclusion of separate heading files.

Reply:
To clarify: we need a new file `toc.md` with all of the values for headings in it AND for the `toc.md` to be listed in this YAML file. Ta